interactions/autocomplete/user/docs.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    name: 'docs',
+    async execute(interaction) {
+        if (interaction.isAutocomplete()) {
+            let focusedOption = interaction.options.getFocused();
+            let choices = require('../../../resources/pages.json').map(keymap => keymap.name);
```

Comment:
keymap?

Reply:
In terms of functionality, it doesn't break anything but I will update it